diff --git a/middle_end/closure/closure.ml b/middle_end/closure/closure.ml
index 4ab577904..cbe34f8d7 100644
--- a/middle_end/closure/closure.ml
+++ b/middle_end/closure/closure.ml
@@ -1053,9 +1053,9 @@ let rec close ({ backend; fenv; cenv ; mutable_vars } as env) lam =
         | Word_size -> make_const_int (8*B.size_int)
         | Int_size -> make_const_int (8*B.size_int - 1)
         | Max_wosize -> make_const_int ((1 lsl ((8*B.size_int) - 10)) - 1 )
-        | Ostype_unix -> make_const_bool (Sys.os_type = "Unix")
-        | Ostype_win32 -> make_const_bool (Sys.os_type = "Win32")
-        | Ostype_cygwin -> make_const_bool (Sys.os_type = "Cygwin")
+        | Ostype_unix -> make_const_bool false
+        | Ostype_win32 -> make_const_bool true
+        | Ostype_cygwin -> make_const_bool false
         | Backend_type ->
             make_const_ptr 0 (* tag 0 is the same as Native here *)
       in
diff --git a/utils/config.mlp b/utils/config.mlp
index 49ffc5bd7..69092561e 100644
--- a/utils/config.mlp
+++ b/utils/config.mlp
@@ -140,11 +140,7 @@ let ext_dll = "%%EXT_DLL%%"
 let host = "%%HOST%%"
 let target = "%%TARGET%%"
 
-let default_executable_name =
-  match Sys.os_type with
-    "Unix" -> "a.out"
-  | "Win32" | "Cygwin" -> "camlprog.exe"
-  | _ -> "camlprog"
+let default_executable_name = "camlprog.exe"
 
 let systhread_supported = %%SYSTHREAD_SUPPORT%%;;
 
@@ -188,7 +184,7 @@ let configuration_variables =
   p "ext_asm" ext_asm;
   p "ext_lib" ext_lib;
   p "ext_dll" ext_dll;
-  p "os_type" Sys.os_type;
+  p "os_type" "Win32";
   p "default_executable_name" default_executable_name;
   p_bool "systhread_supported" systhread_supported;
   p "host" host;
